eox_nelp/course_experience/api/v1/serializers.py

```diff
@@ -1,13 +1,20 @@
 """Serializers used for the experience views."""
+from django.contrib.auth import get_user_model
 from rest_framework_json_api import serializers
 
+from eox_nelp.course_experience.api.v1.relations import ExperienceKeyRelatedField
 from eox_nelp.course_experience.models import (
     FeedbackCourse,
     LikeDislikeCourse,
     LikeDislikeUnit,
     ReportCourse,
     ReportUnit,
 )
+from eox_nelp.edxapp_wrapper.course_overviews import CourseOverview
+
+User = get_user_model()
+COURSE_OVERVIEW_EXTRA_ATTRIBUTES = ["display_name"]
+USER_EXTRA_ATTRIBUTES = ["first_name", "last_name", "fullname", "username"]
```

Comment:
This could be a setting, so we can add an extra attribute whenever we want

Reply:
This could be another PR?